asset/app.scss

```diff
@@ -0,0 +1,69 @@
+@import 'css-reset/reset';
+@import 'fontawesome/css/font-awesome.css';
+$theme-color-1:  #000080;
+$theme-color-2:  #008080;
+html {
+  box-sizing: border-box;
+}
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+
+.timeline {
+    list-style-type: none;
+    border-left: 4px solid $theme-color-1;
+    margin-left: 7em;
+    padding: 1em;
+}
+.event div {
```

Comment:
Do not use raw `div` elements without a class. While raw `div` elements give you a hook for CSS selectors, they do not explain their contents.

Also it makes CSS fragile since `div` elements are so common, then if ANY div is added (regardless of class) the styling will break.